### ISSUE TITLE
Debugger terminates application when executing commands in console while at breakpoint

### ISSUE THREAD

**Issue by mark0978** [pkulchenko/ZeroBraneStudio#627]
Just running the circle.lua turtle sample with a breakpoint on line 6

**Remote Console**
```
Welcome to the interactive Lua interpreter. Enter Lua code and press Enter to run it.
Use Shift-Enter for multiline code.  Use 'clear' to clear the shell output and the history.
Prepend '=' to show complex values on multiple lines. Prepend '!' to force local execution.
print(angle)
```
**Output Tab**
```
Program starting as '"/opt/zbstudio/bin/linux/x64/lua" -e "io.stdout:setvbuf('no')" "/tmp/.8NulGO"'.
Program 'lua' started in '/opt/zbstudio/myprograms' (pid: 11253).
Debugging session started in '/opt/zbstudio/myprograms/'.
Debugging session completed (traced 1 instruction).
Program completed in 18.61 seconds (pid: 11253).
```

**Comment by pkulchenko** [comment]
@mark0978, what version of the IDE are you using? I get `1` printed to the output when I try this. Also, what version of Linux are you running?

**Comment by mark0978** [comment]
RHEL 7.3 and I just installed 1.30 from your website using the shell archive

**Comment by pkulchenko** [comment]
I don't have RHEL, but I have Fedora VM; will test there later today...

**Comment by mark0978** [comment]
I've reproduced this in 14.04.03 as well.

Run the example as above then try to ```print(angle)``` instead of using the watch window.  I was using ```print(angle)``` as a prelude to more complicated statements.  I am aware of the watch window.

I'm also wanting to hook this up to remote debugging which is my next step.

**Comment by pkulchenko** [comment]
@mark0978, this issue is due to a conflict between LuaJIT and wxlua (mostly wxlua issue that has been fixed some time ago); the details are in this thread: http://www.freelists.org/post/luajit/Luajit-wxlua,5.

The fixed version of wxlua will be included as part of the wxwidgets upgrade (#260). You can test it in the [upgrade branch](https://github.com/pkulchenko/ZeroBraneStudio/tree/wxwidgets31x-upgrade).

**Comment by pkulchenko** [comment]
Also, `print` is not very useful in this case; you can achieve the same result in the debugger simply by using those expressions in the console.

**Comment by pkulchenko** [comment]
@mark0978, closing this one as there are no changes needed for this to work; there is a workaround and the fix is available in #260.
